webhook/command_response.go

```diff
@@ -0,0 +1,60 @@
+package webhook
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/micromdm/micromdm/connect"
+	"github.com/micromdm/micromdm/pubsub"
+	"github.com/pkg/errors"
+)
+
+const contentType = "application/x-apple-aspen-mdm"
+
+type commandWebhook struct {
+	Topic string
+	CallbackUrl string
+}
+
+func NewCommandWebhook(topic, callbackURL string) (ListeningWebhook, error) {
```

Comment:
`NewCommandWebhook` should return an exported `Webhook` instead of the interface.